src/cosmos/app/runtime/v1alpha1/module.ts

```diff
@@ -111,20 +111,15 @@ export const Module = {
     return message;
   },
   fromJSON(object: any): Module {
-    return {
-      appName: isSet(object.appName) ? String(object.appName) : "",
-      beginBlockers: Array.isArray(object?.beginBlockers)
-        ? object.beginBlockers.map((e: any) => String(e))
-        : [],
-      endBlockers: Array.isArray(object?.endBlockers) ? object.endBlockers.map((e: any) => String(e)) : [],
-      initGenesis: Array.isArray(object?.initGenesis) ? object.initGenesis.map((e: any) => String(e)) : [],
-      exportGenesis: Array.isArray(object?.exportGenesis)
-        ? object.exportGenesis.map((e: any) => String(e))
-        : [],
-      overrideStoreKeys: Array.isArray(object?.overrideStoreKeys)
-        ? object.overrideStoreKeys.map((e: any) => StoreKeyConfig.fromJSON(e))
-        : [],
-    };
+    const obj = createBaseModule();
+    if (isSet(object.appName)) obj.appName = String(object.appName);
+    if (Array.isArray(object?.beginBlockers)) object.beginBlockers.map((e: any) => String(e));
```

Comment:
Here we are missing an assignment which we had before. The result of the map is lost at the moment.

```suggestion
    if (Array.isArray(object?.beginBlockers)) obj.beginBlockers = object.beginBlockers.map((e: any) => String(e));
```

Reply:
Good catch! Thank you very much!